docs/guides/integration-examples/test-runners.md

```diff
@@ -142,3 +142,66 @@ For AVA there is a [detailed written tutorial](https://github.com/zellwk/ava/blo
 :::note
 Note that this tutorial is pre mongodb-memory-server 7.x.
 :::
+
+## vitest
+
+For [vitest](https://vitest.dev/), create a [global setup file](https://vitest.dev/config/#globalsetup).
+
+`vitest.config.mts`:
+
+```ts
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    globalSetup: ['./globalSetup.ts'],
+  },
+});
+```
+
+`globalSetup.ts`:
+
+```ts
+import type { TestProject } from 'vitest/node';
```

Comment:
Import for `mongodb-memory-server` missing?

Also is the type `TestProject` a global type here?